pkg/operator/k8sutil/configmap.go

```diff
@@ -62,13 +65,28 @@ func GetOperatorSetting(context context.Context, clientset kubernetes.Interface,
 }
 
 func GetValue(data map[string]string, settingName, defaultValue string) string {
```

Comment:
func GetValue(configData map[string]string, settingName, defaultValue string) string {

Reply:
?

Reply:
data -> configData

Reply:
I know it is a small change but I'll prefer to keep it as it is, since it not coming from my change and also not breaking something already existing in code 😄.